### DISCUSSION TITLE
Compatibility Tools Show Unused - Can't Set Games To Use Downloaded Tools

### DISCUSSION THREAD

**Discussion by joel-madero** [DavidoTek/ProtonUp-Qt#471] [General]
Hi All,

Trying to get ProtonUp-QT to work correctly in Manjaro and so far no luck. I have installed ProtonUp-QT 2.10.0. I can install tools fine (I installed GE-Proton9-19). 

Then:
(1) Click Show Game List
+Games listed
(2) Set Resident Evil 2 to GE-Proton9-19
(3) Clicked "click" under ProtonDB
+platinum is shown
(4) Click Apply

Observed:
The change doesn't stick, terminal spits out the following:
`Warning: CompatToolMapping is empty
`

Help appreciated.

**Comment by DavidoTek** [comment]
Hi,

can you set the compatibility tool for Resident Evil from Steam's settings to GE-Proton9-19 and then check if ProtonUp-Qt works correctly after that? (For example, you could change it from GE-Proton9-19 to another compatibility tool and verify that the change from ProtonUp-Qt got applied to Steam.)
Did you just install Steam new on your machine? Steam's configuration files may not be properly initialized after a fresh Steam install. Changing the compatibility tool from Steam's settings should initialize the configuration files.

If that is the case, we should take a look at how to initialize the configuration file from ProtonUp-Qt as the current solution isn't ideal.

> The change doesn't stick, terminal spits out the following:
Warning: CompatToolMapping is empty 

Is there any other output in the terminal? Can you please share the full log.

---

Code reference: https://github.com/DavidoTek/ProtonUp-Qt/blob/d7c71a8e0b6f37a9a1832cd89a8d9c8fdccc6cf0/pupgui2/steamutil.py#L27-L41
